stable/harbor/questions.yaml

```diff
@@ -0,0 +1,577 @@
+  type: enum
+  label: Config JobService JobLogger Type
+  group: "Jobservice Settings"
+  options:
+  - "file"
+  - "database"
+  - "stdout"
+- variable: persistence.persistentVolumeClaim.jobservice.storageClass
+  default: ""
+  description: "If undefined or null, uses the default StorageClass. Default to null"
+  type: storageclass
+  label: Default StorageClass for Jobservice
+  show_if: "persistence.enabled=true&&jobservice.jobLogger=file"
+  group: "Jobservice Settings"
+- variable: persistence.persistentVolumeClaim.jobservice.size
+  required: true
+  default: "1Gi"
+  description: "Jobservice Persistent Volume Size"
+  type: string
+  label: Jobservice Volume Size
+  show_if: "persistence.enabled=true&&jobservice.jobLogger=file"
+  group: "Jobservice Settings"
+- variable: persistence.persistentVolumeClaim.jobservice.existingClaim
+  default: ""
+  description: "If not empty, uses the specified existing PVC instead of creating new one"
+  type: pvc
+  label: Existing Persistent Volume Claim for Jobservice
+  show_if: "persistence.enabled=true&&jobservice.jobLogger=file"
+  group: "Jobservice Settings"
+# database
+- variable: database.type
+  required: true
+  default: "internal"
+  description: "Config Database type, set to internal to deploy a database server as part of this deployment."
+  type: enum
+  label: Config Database Type
+  group: "Database Settings"
+  options:
+  - "internal"
+  - "external"
+- variable: persistence.persistentVolumeClaim.database.storageClass
+  default: ""
+  description: "If undefined or null, uses the default StorageClass. Default to null"
+  type: storageclass
+  label: Default StorageClass for Database
+  show_if: "database.type=internal&&persistence.enabled=true"
+  group: "Database Settings"
+- variable: persistence.persistentVolumeClaim.database.size
+  required: true
+  default: "5Gi"
+  description: "Database Persistent Volume Size"
+  type: string
+  label: Database Volume Size
+  group: "Database Settings"
+  show_if: "database.type=internal&&persistence.enabled=true"
+- variable: persistence.persistentVolumeClaim.database.existingClaim
+  default: ""
+  description: "If not empty, uses the specified existing PVC instead of creating new one"
+  type: pvc
+  label: Existing Persistent Volume Claim for Database
+  group: "Database Settings"
+  show_if: "database.type=internal&&persistence.enabled=true"
+- variable: database.external.host
+  required: true
+  default: ""
+  description: "Host of the external database"
+  type: string
+  label: External Database Host
+  show_if: "database.type=external"
+  group: "Database Settings"
+- variable: database.external.username
+  required: true
+  default: ""
+  description: "Existing username in the external DB"
+  type: string
+  label: External Database username
+  show_if: "database.type=external"
+  group: "Database Settings"
+- variable: database.external.password
+  default: ""
+  required: true
+  description: "External database password"
```

Comment:
It sounds obvious, but maybe you want to mention that the password is related to "existing username"?

(I was probably bit puzzled by the wording for username)

Reply:
done